.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: Test
+on: [push]
```

Comment:
Also run the test for other conditions like `pull_request, workflow_dispatch`

Reply:
Done.